homework/content.js

```diff
@@ -1,37 +1,39 @@
 'use strict';
 
-(function() {
-  var _wrapper = document.querySelector('#note-content-wrapper');
+function ToDoItemReader() {
+  this._wrapper = document.querySelector('#note-content-wrapper');
+}
+ToDoItemReader.prototype = {
+    start: function() {
+	window.addEventListener('note-open', function(event) {
```

Comment:
Instead of using an anonymous function, set the listener as `this` and handle events with `handleEvent` method is better.